website/layouts/index.html

```diff
@@ -55,15 +55,21 @@
         <div class="blog-excerpts">
           <h2>Recently Published</h2>
           <article class="doc">
-            {{ $paginator := .Paginate (where .Site.RegularPages "Type" "in" .Site.Params.mainSections) }}
+            {{ $paginator := .Paginate (where .Site.RegularPages "Type" "in" .Site.Params.mainSections) 2 }}
             {{ range ($paginator.Pages) }}
               {{ partial "post-preview.html" . }}
             {{ end }}
 
+            {{/* This div is not really for pagination but the CSS does what we want and also I'm lazy */}}
+            <div class="pagination-centered">
+              <div class="pagination pagination-default" style="padding: 3em 0; opacity: 50%;">
+                {{ range .Site.Menus.main }}{{ if eq .Name "Blog" }}
+                  <a href="{{.URL}}">More articles &hellip;</a>
+                {{ end }}{{ end }}
+              </div>
+            </div>
+
```

Comment:
Could be simpler, something like might work:

```suggestion
            {{ $posts := where .Site.RegularPages "Type" "in" .Site.Params.mainSections }}
            {{ range $posts.Limit 2 }}
              {{ partial "post-preview.html" . }}
            {{ end }}

            {{/* This div is not really for pagination but the CSS does what we want and also I'm lazy */}}
            <div class="pagination-centered">
              <div class="pagination pagination-default" style="padding: 3em 0; opacity: 50%;">
                <a href="/posts">More articles &hellip;</a>
              </div>
            </div>

```

Reply:
Re `<a href="/posts">More articles &hellip;</a>` I was trying to avoid hard coding the url..

Reply:
Included in latest revision.

Reply:
Thanks!